### DISCUSSION TITLE
[GENERAL] VPS Choice Help or How many players can this VPS handle while hosting only WoW WoTLK 3.3.5a using AzerothCore?

### DISCUSSION THREAD

**Discussion by DivineParadox** [azerothcore/azerothcore-wotlk#3891] [General]
Hello,

I intend to open a World of Warcraft: Wrath of The Lich King Server using AzerothCore. Even though I have some technical experience, I don't have experience with World of Warcraft Private Servers, therefore I have to ask how many players can those VPS handle and which should I choose? The most obvious one to me is the second option.

Regarding operating system, I was thinking about using Linux, as it uses the least resources and is much safer compared to Windows.

The server is intended to have a few modules and the targeted player base is mainly EU.

The first offer I received is for €100/month with the following specifications: [Hosted in the US]
- CPU Cores: 7
- Processor: Intel Xeon E-2288G @ 3.70GHz 
- DDR4 RAM: 16GB
- Storage: 200GB SSD
- Download Speed: 10Gbps
- Upload Speed: 1Gbps
- Public IP Address: 1
- IOPS: 7500
- Automatic Backup: No
- Operating System: Any.
- DDoS Protection: Add-On for €20
- SSL Certificate: No
- Server Management: No
- Technical Assistance: Connectivity Issues only

The second offer I've received is for €118/month with the following specifications: [Hosted in the EU]
- CPU Cores: 8
- Processor: Intel Xeon E-2288G @ 3.70GHz 
- Memory: 8GB RAM & 16GB vSWAP
- Storage: 250GB SSD
- Download Speed: 10Gbps
- Upload Speed: 1Gbps
- Dedicated IP: IPv4 & IPv6
- IOPS: 16000
- Automatic Backup: Yes
- Operating System: Linux
- SSL Certificate: Yes
- Server Management: Yes
- Technical Assistance: 24/7

Thank you for the help!

Best regards,
Leventhe

**Comment by FrancescoBorzi** [comment]
If your player base is in Europe, then it's better to have your server hosted in Europe. Also AzerothCore requires 16 GB of RAM to run fine in production. I'm not sure how the `8GB RAM & 16GB vSWAP` RAM works so I can't really give an opinion here.

**Comment by DivineParadox** [comment]
Hello FrancescoBorzi,

I've talked to a representative of the Hosting Company that made the second offer via Live Chat. They made a new offer, which is as following:

Price: €160/month
CPU Cores: 16
Processor: Intel XEON E5 @ 3.60GHz 
Memory: 16GB RAM
Storage: 250GB SSD
Download Speed: 10Gbps
Upload Speed: 1Gbps
Dedicated IP: IPv4 & IPv6
IOPS: 16000
Automatic Backup: Yes
Operating System: Linux
SSL Certificate: Yes
Server Management: Yes
Technical Assistance: 24/7

Can you review this version? It is not possible to have more than 16 GB RAM.

Thank you for your fast reply!

Best regards,
Leventhe

**Comment by FrancescoBorzi** [comment]
@andreimauta that should be good

**Comment by BRNKR** [comment]
@andreimauta First question you should have. How do you want to get so much player who will get your silicon hot?

What about this for the beginning?
https://www.netcup.de/bestellen/produkt.php?produkt=2631

**Comment by Si1ker** [comment]
16 GB if you're running in release for 200 players = ok
Recommend get 32 GB RAM for other build than release and 200+ players
ac-prod: cpu < ram

**Comment by LichKing255** [comment]
> 16 GB if you're running in release for 200 players = ok
> Recommend get 32 GB RAM for other build than release and 200+ players
> ac-prod: cpu < ram

How much RAM does warmane use for its players?

**Comment by Si1ker** [comment]
@LichKing255 Go ask them.
